app/src/main/java/com/willowtree/vocable/presets/PresetsRepository.kt

```diff
@@ -159,34 +159,28 @@ class PresetsRepository(context: Context) : KoinComponent {
         val mySayingsCategory =
             categoryObjects.first { it.categoryId == mySayingsCategoryId }
         val mySayings = sharedPrefs.getMySayings()
-        if (mySayings.isNotEmpty()) {
-            mySayings.forEach {
-                val phraseId = UUID.randomUUID().toString()
-                phraseObjects.add(
-                    Phrase(
-                        phraseId,
-                        System.currentTimeMillis(),
-                        true,
-                        System.currentTimeMillis(),
-                        mapOf(Pair(Locale.US.language, it)),
-                        phraseObjects.size
-                    )
+        mySayings.forEach {
+            val phraseId = UUID.randomUUID().toString()
+            phraseObjects.add(
+                Phrase(
+                    phraseId,
+                    System.currentTimeMillis(),
+                    true,
+                    System.currentTimeMillis(),
+                    mapOf(Pair(Locale.US.language, it)),
+                    phraseObjects.size
                 )
-                crossRefObjects.add(
-                    CategoryPhraseCrossRef(
-                        mySayingsCategory.categoryId,
-                        phraseId
-                    )
+            )
+            crossRefObjects.add(
+                CategoryPhraseCrossRef(
+                    mySayingsCategory.categoryId,
+                    phraseId
                 )
-            }
-            sharedPrefs.setMySayings(emptySet())
-        } else {
-            val phrases = getPhrasesForCategory(mySayingsCategoryId)
-            if (phrases.isEmpty()) {
-                mySayingsCategory.hidden = true
-            }
+            )
         }
 
+        sharedPrefs.setMySayings(emptySet())
```

Comment:
So I updated the places where we were setting the My Sayings category to hidden in the `edit-categories-implementation` branch. Might want to make sure there's no conflicts with that.